app-crypt/moolticute/files/moolticuted.init

```diff
@@ -0,0 +1,18 @@
+#!/sbin/openrc-run
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+command=/usr/bin/moolticuted
+command_args=""
+pidfile=/var/run/moolticuted.pid
+
+depend() {
+	need local udev
+}
+
+start() {
```

Comment:
Why do you need to redefine it? It looks pretty standard to me.

Reply:
Oh, I found the `start_stop_daemon_args` to place required `--make-pidfile` and `--background` params!